src/open_samus_returns_rando/files/levels/s110_surfaceb.lua

```diff
@@ -560,7 +560,7 @@ function s110_surfaceb.OnSubAreaChange(_ARG_0_, _ARG_1_, _ARG_2_, _ARG_3_, _ARG_
   Game.SetScenarioItemEnabledByName("ray01", false)
   Game.SetSceneGroupEnabledByName("sg_debris02", false)
   Game.SetSceneGroupEnabledByName("sg_debris03", false)
-  if _ARG_2_ == "collision_camera_000" and Game.GetItemAmount(Game.GetPlayerName(), "ITEM_ADN") < 39 then
+  if _ARG_2_ == "collision_camera_000" and Blackboard.GetProp("GAME", "OBJECTIVE_COMPLETE") == false then
```

Comment:
We usually don't do explicit checks for booleans. Dunno if you want to keep it.

Reply:
Ah right. This would normally be `not blah`. I'll fix it.